### ISSUE TITLE
Obtain all possible formats

### ISSUE THREAD

**Issue by mcrodrig** [gr2m/moment-parseformat#108]
Hi,

I need to obtain the common format for an array of dates/time. As I understand the library, this only returns the first occurency based on the options.

Is it possible to obtain all the format for a single date/time? For example:
07-05-2012 -> ["dd-MM-YYYY", "MM-dd-YYYY"]

Thanks,

**Comment by gr2m** [comment]
It’s not currently possible

**Comment by mcrodrig** [comment]
Any plan in the short term?

**Comment by gr2m** [comment]
No, sorry. It might be an interesting addition, as long as it doesn't impact the package size too much. I'd be happy to review a PR, but I don't plan to add it myself
